coreneuron/io/reports/report_handler.cpp

```diff
@@ -253,6 +263,86 @@ VarsToReport ReportHandler::get_section_vars_to_report(const NrnThread& nt,
     return vars_to_report;
 }
 
+VarsToReport ReportHandler::get_summation_vars_to_report(
+    const NrnThread& nt,
+    const std::set<int>& target,
+    ReportConfiguration& report,
+    const std::vector<int>& nodes_to_gids) const {
+    VarsToReport vars_to_report;
+    const auto* mapinfo = static_cast<NrnThreadMappingInfo*>(nt.mapping);
+    auto* alu_mapping = static_cast<ALUMapping*>(nt.alu_);
+    auto& alu = alu_mapping->report_ALU_[report.output_path];
+    if (!mapinfo) {
+        std::cerr << "[COMPARTMENTS] Error : mapping information is missing for a Cell group "
+                  << nt.ncell << '\n';
+        nrn_abort(1);
+    }
+    for (int i = 0; i < nt.ncell; i++) {
+        int gid = nt.presyns[i].gid_;
+        if (report.target.find(gid) == report.target.end()) {
+            continue;
+        }
+        bool has_imembrane = false;
+        int scalar = 1;
```

Comment:
nitpick : what the scalar mean here?

Reply:
@jorblancoa : you marked this resolved but with the comment I still found it confusing :)
So, `scalar` means if we need to `scale` because of units? (in that case `scale` and `scalar` are different things)

Reply:
I can rename it to scale if you prefer, I was just following neurodamus naming (since it is copying the behavior)